src/test/java/org/lobid/resources/Hbz01MabXml2ElasticsearchLobidTest.java

```diff
@@ -272,16 +273,23 @@ static private SearchResponse getElasticsearchDocuments() {
 		static String getAsNtriples() {
 			return Arrays.asList(getElasticsearchDocuments().getHits().getHits())
 					.parallelStream()
-					.flatMap(hit -> Stream.of(toRdf(hit.getSourceAsString())))
+					.flatMap(
+							hit -> Stream.of(toRdf(cleanseEndtime(hit.getSourceAsString()))))
```

Comment:
Oh, `Stream.of` is cool, didn't know that. But in this case here, it seems we could use `map` and avoid the stream altogether, like this: `.map(hit -> toRdf(cleanseEndtime(hit.getSourceAsString())))`.

Reply:
Admittedly not aware of the difference. Using the simpler one as you propose.